src/wechaty-mixins/puppet-mixin.ts

```diff
@@ -545,6 +637,10 @@ const puppetMixin = <MixinBase extends WechatifyUserModuleMixin & GErrorMixin &
                   case PUPPET.types.Payload.Message:
                     // Message does not need to dirty (?)
                     break
+                  case PUPPET.types.Payload.Tag:
```

Comment:
Add `TODO` here or not?

Reply:
I'm not sure about your intention here...

Reply:
Something need to do when receive `Tag` or `TagGroup` dirty event or not?